tools/CodeGenerator/CodeGenerator.csproj

```diff
@@ -1,15 +1,21 @@
-<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
     <OutputType>Exe</OutputType>
+    <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="1.2.0-*" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="1.2.0-*" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <StartWorkingDirectory>$(MSBuildThisFileDirectory)..\..\src\Microsoft.AspNetCore.Server.Kestrel\Internal\Http</StartWorkingDirectory>
+    <StartArguments>FrameHeaders.Generated.cs Frame.Generated.cs</StartArguments>
+  </PropertyGroup>
```

Comment:
It is always run with those arguments. This makes it more convenient to re-generate our code-generated classes by just running `dotnet run` in `tools/CodeGenerator`, without having to pass it any arguments.